### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by hohoDigidea**
#### Summary

Added a note from https://www.w3.org/TR/css-font-loading-3/#font-face-load

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/fontface/load/index.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ browser-compat: api.FontFace.load
 
 The **`load()`** method of the {{domxref("FontFace")}} interface loads a font based on current object's constructor-passed requirements, including a location or source buffer, and returns a {{jsxref('Promise')}} that resolves with the current `FontFace` object.
 
+> **Note:** The **`load()`** method of `FontFace` forces a url-based font face to request its font data and load. For fonts constructed from binary data, or fonts that are already loading or loaded, it does nothing.
+
```
Comment (wbamberg) [inline files/en-us/web/api/fontface/load/index.md:20]: This bit seems strange now: "loads a font based on current object's constructor-passed requirements, including a location or source buffer" - "source buffer" I suppose means binary data, but then in the next paragraph we say it **does nothing** if it was given binary data.

What do you think about:

```suggestion
The **`load()`** method of the {{domxref("FontFace")}} interface requests and loads a font whose `source` was specified as a URL. It returns a {{jsxref('Promise')}} that resolves with the current `FontFace` object.

If the `source` for the font face was specified  as binary data, or the font {{domxref("FontFace/status", "status")}} property of the font face is anything other than `unloaded`, then this method does nothing.
```

Does that seem accurate to you?

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @LIXiangChen ! I had a suggestion, what do you think?

File: files/en-us/web/api/fontface/load/index.md (hunk 1 above)
Comment (LIXiangChen) [reply files/en-us/web/api/fontface/load/index.md]: Of course no problem. I didn't know if I had the right to make changes to the original text, so I simply added a paragraph copied from the W3C.

Comment (wbamberg) [reply files/en-us/web/api/fontface/load/index.md]: You can absolutely suggest changes to the text, we do it all the time :).

File: files/en-us/web/api/fontface/load/index.md [hunk 2]
```diff
@@ -14,8 +14,9 @@ browser-compat: api.FontFace.load
 ---
 {{APIRef("CSS Font Loading API")}}
 
-The **`load()`** method of the {{domxref("FontFace")}} interface loads a font based on current object's constructor-passed requirements, including a location or source buffer, and returns a {{jsxref('Promise')}} that resolves with the current `FontFace` object.
+The **`load()`** method of the {{domxref("FontFace")}} interface requests and loads a font whose `source` was specified as a URL. It returns a {{jsxref('Promise')}} that resolves with the current `FontFace` object.
 
+If the `source` for the font face was specified  as binary data, or the font {{domxref("FontFace/status", "status")}} property of the font face is anything other than `unloaded`, then this method does nothing.
 ## Syntax
```
Comment (wbamberg) [inline files/en-us/web/api/fontface/load/index.md:20]:
```suggestion
If the `source` for the font face was specified  as binary data, or the font {{domxref("FontFace/status", "status")}} property of the font face is anything other than `unloaded`, then this method does nothing.

## Syntax
```

**Review by wbamberg — APPROVED** [review]
👍 thank you, @LIXiangChen !

**Pull request merged**